collect_app/src/main/java/org/odk/collect/android/activities/MainMenuActivity.java

```diff
@@ -648,4 +654,13 @@ public void onChange(boolean selfChange) {
         }
     }
 
+    private void showSMSFeatureDisabledDialogIfNeeded() {
```

Comment:
Nit: Can we rename to e.g. `disableSmsIfNeeded` to clarify that this has side-effects?